TAO/TAO_IDL/be/be_helper.cpp

```diff
@@ -597,7 +612,7 @@ TAO_OutStream::print (AST_Expression *expr)
       this->TAO_OutStream::print ("L'%lc'", ev->u.wcval);
       break;
     case AST_Expression::EV_octet:
-      this->TAO_OutStream::print ("%d", ev->u.oval);
+      this->TAO_OutStream::print ("0x%x", ev->u.oval);
```

Comment:
Could you tell me how would it be breaking?

Reply:
I'm not sure where this is used, so that's what I was asking the question.  If it's generating literals for the C++ output then I don't think it's going to matter.

Reply:
This should only be used for C++ output.